algosdk/template.py

```diff
@@ -174,6 +183,40 @@ def get_program(self):
         types = [int, int, "address", "base64", "address", int]
         return inject(orig, offsets, values, types)
 
+    @staticmethod
+    def get_transaction(contract, preimage, first_valid, last_valid, gh, fee):
+        """
+        Return a group transactions array which transfer funds according to
+        the contract's ratio.
```

Comment:
Wrong comment `Return a transaction which will release funds if a matching preimage is used.`